### PR TITLE
Update vNext changelog to RC

### PR DESCRIPTION
**Pull request by desistefanova**

### CODE REVIEW

File: .github/workflows/publish-release.yml [hunk 1]
```diff
@@ -233,7 +233,7 @@ jobs:
       run: |
         echo "## vNext (TBD)
 
-        **This project is in the Beta stage. The API should be quite stable, but occasional breaking changes may be made.**
+        **This project is in Release Candidate stage.**
```
Comment (nielsenko) [inline .github/workflows/publish-release.yml:236]:
```suggestion
        **This project is in the Release Candidate stage.**
```
perhaps?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## vNext (TBD)
 
-**This project is in the Beta stage. The API should be quite stable, but occasional breaking changes may be made.**
+**This project is in Release Candidate stage.**
```
Comment (nielsenko) [inline CHANGELOG.md:3]: same

File: .github/workflows/publish-release.yml (hunk 1 above)
Comment (desistefanova) [reply .github/workflows/publish-release.yml:236]: I copied the text from what we already released https://pub.dev/packages/realm. I think both are acceptable.

Comment (blagoev) [reply .github/workflows/publish-release.yml:236]: Not a native English speaket, but the current text seems ok to me

**Review by blagoev — APPROVED** [review]

**Pull request merged**